src/views/Viewer.vue

```diff
@@ -25,8 +25,9 @@
 	<Modal v-if="initiated || currentFile.modal"
 		id="viewer"
 		size="full"
-		:class="{'icon-loading': !currentFile.loaded && !currentFile.failed,
-			'theme--undefined': theme === null, 'theme--dark': theme === 'dark', 'theme--light': theme === 'light', 'theme--default': theme === 'default'}"
+		:class="[{'icon-loading': !currentFile.loaded && !currentFile.failed,
+			'theme--undefined': theme === null, 'theme--dark': theme === 'dark',
+			'theme--light': theme === 'light', 'theme--default': theme === 'default'}, `app-${handlerId}`]"
```

Comment:
I'd prefer to set a data attribute like `:data-handler="handlerId"` as the handler id might also contain characters that are not supported in css selector classes

Reply:
I agree, it will be more future-proof and prevents CSS conflicts.